src/Setup.php

```diff
@@ -377,5 +426,39 @@ public function bcgov_block_theme_acf_init_block_types() {
 			}
 		}
 	}
+
+	/**
+     * WordPress filter hook for headers.
+     *
+     * @param array $headers  An array of headers.
+     * @return array
+     */
+    public function bcgov_block_theme_wp_headers( $headers ) {
+        $csp         = '';
```

Comment:
as a consideration, depending on the hit this might take, is to have one option and serialize the data.  Unfortunately this would be more work.  So hopefully this isn't an issue.
Also would have like to see the options add to the default values instead of replace.

Reply:
Default values added. get_option caches behind the scenes, so hopefully this proves a workable solution.